### PR TITLE
fix: cryptsetup remove slot

### PR DESCRIPTION
**Pull request by frezbo**
Fix cryptsetup remove slot operation.

Signed-off-by: Noel Georgi <git@frezbo.dev>

### CODE REVIEW

**Comment by frezbo** [comment]
Build passed here: https://ci.dev.talos-systems.io/siderolabs/talos/29284/1/5

Test also passes locally.

```bash
❯ go test -exec sudo -v --count 1 -run ^TestLUKSSuite\$ github.com/talos-systems/go-blockdevice/blockdevice/encryption/luks
[sudo] password for frezbo: 
=== RUN   TestLUKSSuite
=== RUN   TestLUKSSuite/TestEncrypt
    test.go:38: Using /dev/loop0
    luks_test.go:77: unencrypted partition path /dev/loop0p2
    test.go:56: Freeing /dev/loop0
--- PASS: TestLUKSSuite (9.26s)
    --- PASS: TestLUKSSuite/TestEncrypt (9.25s)
PASS
ok      github.com/talos-systems/go-blockdevice/blockdevice/encryption/luks     14.415s
```

File: blockdevice/encryption/luks/luks.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ func (l *LUKS) CheckKey(devname string, key *encryption.Key) (bool, error) {
 
 // RemoveKey adds a new key at the LUKS encryption slot.
```
Comment (smira) [inline blockdevice/encryption/luks/luks.go:244]: looks the comment is not in sync

**Review by smira — APPROVED** [review]

File: blockdevice/encryption/luks/luks.go (hunk 1 above)
Comment (frezbo) [reply blockdevice/encryption/luks/luks.go]: fixed

**Pull request merged**